recipes/dssp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "3.0.0" %}
+{% set name = "dssp" %}
+# This file, the patch file and build file were taken from conda salilab channel's recipes - please see https://github.com/salilab/conda-recipes for more.
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://salilab.org/dssp/dssp-3.0.0.tgz
+  sha256: 25d39a2107767b622a59dd9fa790112c1516564b8518fc6ac0fab081d5ac3ab0
+  patches:
+    - dssp-boost-tr1.patch #Rebuild with latest gcc and boost - this was taken from the 
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('dssp', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - boost-cpp
```

Comment:
Only 'make' and the compilers should be in 'build'. The libraries should be in 'host'